nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -228,6 +228,16 @@ in
                 The hostname of the build machine.
               '';
             };
+            protocol = mkOption {
+              type = types.nullOr types.str;
```

Comment:
On second thought, should this be `types.enum [ "ssh" "ssh-ng" ]` instead?

Reply:
Ah, I didn't know about it.